benchmark/net/chunky_http_client.js

```diff
@@ -0,0 +1,104 @@
+"use strict";
+// test HTTP throughput in fragmented header case
+var common = require('../common.js');
+var net = require('net');
+var test = require('../../test/common.js');
+
+var bench = common.createBenchmark(main, {
+//  len: [1, 4, 8, 16, 32, 64, 128],
+  len: [1],
+  num: [5, 50, 500, 2000],
+  type: ['send'],
+}
+)
+
+
+function main(conf) {
+  var len;
+  var num;
+  var type;
+  num = +conf.num;
+  len = +conf.len;
+  type = conf.type;
```

Comment:
Style: declaration and assignment normally go on the same line.  You can use let or const if you want.